src/sentry/tagstore/snuba/backend.py

```diff
@@ -248,8 +248,8 @@ def __get_tag_keys_for_projects(
         keys=None,
         include_values_seen=True,
         use_cache=False,
-        include_transactions=False,
```

Comment:
Removing include_transactions parameter here because it is redundant and strictly less customizable than the new dataset param. I have updated all usages of `__get_tag_keys_for_projects` to use the correct dataset as a result, feel free to double check.